cds/modules/deposit/api.py

```diff
@@ -221,6 +221,7 @@ def commit(self, **kwargs):
         """Set partial validator as default."""
         if 'validator' not in kwargs:
             kwargs['validator'] = PartialDraft4Validator
+        # TODO add 'modified_by' inside _cds
```

Comment:
Ok, shall we create an issue or shall we keep the TODO? I do not know how you guys proceed in this cases? What do you suggest?